src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -2154,32 +2159,34 @@ static async Task ExecuteAwaited(ValueTask task)
 
     private static ValueTask<object?> ExecuteTaskWithEmptyResult(Task task)
     {
+        Debug.Assert(EmptyHttpResultInstance is not null);
```

Comment:
Is not possible to it be `null` in the `SignalR` scenario?

Reply:
Yep, the problem in the SignalR tests was the resolution that was happening for the assembly in the static properties. I cleaned this up to account for Stephen's feedback and reused the same `EmptyHttpResult.Instance` variable when constructing the property.